tf2onnx/custom_opsets/onnx_ml.py

```diff
@@ -42,10 +42,16 @@ def version_8(cls, ctx, node, initialized_tables, **kwargs):
 
         dtype = ctx.get_dtype(node.output[0])
         in_dtype = ctx.get_dtype(node.input[1])
-        utils.make_sure(dtype == TensorProto.INT64 and in_dtype == TensorProto.STRING,
-                        "Only lookup tables of type string->int64 are currently supported.")
+        utils.make_sure((dtype == TensorProto.INT64 and in_dtype == TensorProto.STRING or \
+                         dtype == TensorProto.STRING and in_dtype == TensorProto.INT64),
+                        "Only lookup tables of type string->int64 or int64->string are currently supported.")
 
-        cats_strings, cats_int64s = initialized_tables[shared_name]
+        if in_dtype == TensorProto.STRING:
```

Comment:
Hi @rmarrugat , Thanks for your contribution.
This PR is been over half of a year. Would you mind create a new PR and add the unit test for it?